about/understanding-openshift-pipelines.adoc

```diff
@@ -35,6 +35,13 @@ include::modules/op-about-taskrun.adoc[leveloffset=+2]
 include::modules/op-about-pipelines.adoc[leveloffset=+2]
 //About pipeline run
 include::modules/op-about-pipelinerun.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Needs an anchor ID. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies